src/quest_manager/templatetags/quest_tags.py

```diff
@@ -14,3 +16,33 @@ def is_hidden(quest, user):
     if not user or not quest:
         return None
     return user.profile.is_quest_hidden(quest)
+
+
+@register.simple_tag(takes_context=True)
+def can_export(context):
```

Comment:
What's the reason for doing this in the template instead of passing it as a context variable in the view?